superset/utils/core.py

```diff
@@ -1326,6 +1326,11 @@ def get_metric_names(metrics: Sequence[Metric]) -> List[str]:
     return [metric for metric in map(get_metric_name, metrics) if metric]
 
 
+def get_first_metric_name(metrics: Sequence[Metric]) -> Optional[str]:
```

Comment:
this should probably be `get_first_metric_label`, but i'll stamp because it's only a revert

Reply:
Although @etr2460 the function call `get_metric_names` but assigns these to the `metric_labels` variable thus the variable as opposed to the function could be misnamed.